app/routes/planet_routes.py

```diff
@@ -0,0 +1,59 @@
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
+from .helpers import validate_planet
+
+planet_bp = Blueprint("planet", __name__, url_prefix="/planets")
+
+@planet_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+
+    new_planet = Planet.create(request_body)
+
+    db.session.add(new_planet)
+    db.session.commit()
+
+    return make_response(jsonify(f"Planet {new_planet.name} has been created"), 201)
+
+@ planet_bp.route("", methods=["GET"])
+def get_planets():
+    planet_query = request.args.get("name")
+    color_query = request.args.get("color")
+    if planet_query:
+        planets = Planet.query.filter_by(name=planet_query)
+    elif color_query:
+        planets = Planet.query.filter_by(color=color_query)
+    else:
+        planets = Planet.query.all()
+
+    planets_response = [planet.to_json() for planet in planets]
+
+    return make_response(jsonify(planets_response), 200)
```

Comment:
Nice work with filtering and using list comprehension! The comment about not needing `make_response` also applies to this view function.